transport/src/main/java/io/netty/channel/DefaultChannelId.java

```diff
@@ -105,11 +106,33 @@ public static DefaultChannelId newInstance() {
         MACHINE_ID = machineId;
     }
 
-    private static int defaultProcessId() {
-        ClassLoader loader = null;
+    static int processHandlePid(ClassLoader loader) {
+        /*pid is positive on unix, non{-1,0} on windows*/
+        int nilValue = -1;
+        if (PlatformDependent.javaVersion() >= 9) {
+            Long pid;
+            try {
+                Class<?> processHandleImplType = Class.forName("java.lang.ProcessHandleImpl", true, loader);
+                Method processHandleCurrent = processHandleImplType.getMethod("current");
+                processHandleCurrent.setAccessible(true);
```

Comment:
These methods are public. Do we actually need to make them accessible?

Reply:
setAccessible() *was* needed because `java.lang.ProcessHandleImpl` is package-private. Switched to `lava.lang.ProcessHandle` and removed setAccessible().